fhir-persistence/src/main/java/com/ibm/fhir/persistence/MultiResourceResult.java

```diff
@@ -44,15 +43,16 @@ private MultiResourceResult(Builder<T> builder) {
     public boolean isSuccess() {
         return success;
     }
+
     /**
-     * The resources returned from the interaction
-     * 
+     * The resource results returned from the interaction
      * @return
-     *     An unmodifiable list containing immutable objects of type {@link Resource}.
+     *     An unmodifiable list containing immutable objects of type {@link ResourceResource<T>}
```

Comment:
```suggestion
     *     An unmodifiable list containing immutable objects of type {@link ResourceResult<T>}
```

Reply:
also curious:  does the link work when you use a generic type like that?

Reply:
Yes, works with a generic (in eclipse, at least). Fixed.